pkg/cmd/list.go

```diff
@@ -60,7 +60,12 @@ func listVolumes(cmd *cobra.Command, args []string, out io.Writer) error {
 		if err != nil {
 			return err
 		}
-		if _, err := fmt.Fprintf(writer, "%s%s\t%s\t\n", prefix, volume.Name, branch); err != nil {
+		containers, err := dvol.ContainerRuntime.Related(volume.Name)
```

Comment:
I don't like reaching into APIs. I think `ContainerRuntime` should be private and this call should be `dvol.relatedContainers(volume.Name)`.